actor/builtin/miner/miner.go

```diff
@@ -516,15 +516,23 @@ func (ma *Actor) GetPoStState(ctx exec.VMContext) (*big.Int, uint8, error) {
 	return big.NewInt(result), 0, nil
 }
 
-// GetSectorCommitments returns all sector commitments posted by this miner.
-func (ma *Actor) GetSectorCommitments(ctx exec.VMContext) (map[string]types.Commitments, uint8, error) {
+// GetProvingSetCommitments returns all sector commitments posted by this miner.
+func (ma *Actor) GetProvingSetCommitments(ctx exec.VMContext) (map[string]types.Commitments, uint8, error) {
 	if err := ctx.Charge(actor.DefaultGasCost); err != nil {
 		return nil, exec.ErrInsufficientGas, errors.RevertErrorWrap(err, "Insufficient gas")
 	}
 
 	var state State
 	out, err := actor.WithState(ctx, &state, func() (interface{}, error) {
-		return (map[string]types.Commitments)(state.SectorCommitments), nil
+		commitments := NewSectorSet()
```

Comment:
FYI consider `ReadState()` for more concise read-only methods (saves you some type assertions later too)

Reply:
done.